packages/core/src/text/Text.css

```diff
@@ -1,7 +1,7 @@
 /* Main css class. Style for body text */
 .uitkText {
-  font-size: var(--uitk-text-fontSize);
-  line-height: var(--uitk-text-lineHeight);
+  font-size: var(--uitkText-fontSize, var(--uitk-text-fontSize));
```

Comment:
Do we want public tokens for every variant? I think we should follow [Button](https://github.com/jpmorganchase/uitk/blob/main/packages/core/src/button/Button.css#L86) and have a "global" override that can be scoped with classes vs a per variant solution